cockroachcloud/serverless-cluster-management.md

```diff
@@ -49,18 +49,18 @@ The monthly cost estimate is calculated using simple extrapolation that assumes
 1. In the **Usage this month** section of your cluster's [**Overview** page](cluster-overview-page.html), click **Estimate usage cost**.
 1. Select a time frame in which your workload was active.
 
-    Your used [RUs](learn-about-request-units.html), used storage, and spend from the selected time frame will be shown along with your monthly cost estimate. The spend and cost estimate do not account for the [free resources](learn-about-pricing.html#free-vs-paid-usage) granted to every {{ site.data.products.serverless }} cluster, which you would have to use up before being charged.
+    Your used [RUs](learn-about-request-units.html), used storage, and spend from the selected time frame will be shown along with your monthly cost estimate. The spend and cost estimate do not account for the [free resources](learn-about-pricing.html#free-vs-paid-usage) granted to every organization, which you would have to use up before being charged.
 
-## Edit your spend limit
+## Edit your resource limits
 
-You can edit your spend limit from the **Overview** page. The change in your spend limit will affect current and upcoming billing cycles. See [Learn about {{ site.data.products.serverless }} Pricing](learn-about-pricing.html) for more information on resource usage.
+You can edit your resource limits from the **Overview** page. The change in your resource limits will affect current and upcoming billing cycles. See [Planning your cluster](plan-your-cluster.html) for more information on resource usage.
```

Comment:
Suggest linking "resource limits" to the definition of that term from _includes/common/basic-terms.md (whichever page has that include and makes sense)